app/templates/services/edit_submission_section.html

```diff
@@ -1,35 +1,42 @@
 {% extends "services/_base_edit_section_page.html" %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {% with items = [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
       },
       {
-        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-        "label": "Apply to " + framework.name
+        "text": "Apply to " + framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_submission_lots", framework_slug=framework.slug),
-        "label": "Services"
+        "text": lot.name,
+        "href": url_for(".framework_submission_lots", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot),
-        "label": service_data.lotName
-      },
-      {
-        "link": url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id),
-        "label": service_data.get('serviceName', service_data['lotName'])
-      } if not one_service_limit else {}
+        "text": "Services",
+        "href": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot)
+      }
     ]
   %}
-    {% include "toolkit/breadcrumb.html" %}
+    {% if lot.oneServiceLimit %}
+      {{ govukBreadcrumbs({"items": items + [{"text": section.name}]}) }}
+    {% else %}
+      {{ govukBreadcrumbs({"items": items + [
+        {
+          "text": service_data.get('serviceName', service_data['lotName']),
+          "href": url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id)
+        },
+        {
+          "text": section.name
+        }
+      ]}) }}
```

Comment:
This isn't working for me :(

With DOS4 open for applications when I visit http://localhost/suppliers/frameworks/digital-outcomes-and-specialists-4/submissions/digital-outcomes/150182 I see this:

![image](https://user-images.githubusercontent.com/503614/71190560-b8352080-227c-11ea-8c73-904e4dae0811.png)

Reply:
Did you update the framework's `applications_close_at_utc` to be in the future or only the `status` to `open`?

Reply:
Oh interesting, I only changed the status

Reply:
Changing `applications_close_at_utc` doesn't seem to have changed anything

Reply:
Expanded the condition of line 27 to check if we create or update the service to: 
`{% if lot.oneServiceLimit or 'serviceName' not in service_data %}`
The former was causing the problem.